src/NJsonSchema.CodeGeneration/DefaultEnumNameGenerator.cs

```diff
@@ -30,11 +30,15 @@ public string Generate(int index, string name, object value, JsonSchema4 schema)
             }
 
             return ConversionUtilities.ConvertToUpperCamelCase(name
-                .Replace(":", "-").Replace(@"""", @""), true)
+                .Replace(":", "_").Replace(@"""", @""), true)

```

Comment:
I think its important to keep this "-" (inside ConvertToUpperCamelCase) as for example foo-bar will be transformed to FooBar and foo_bar will probably be transformed to Foo_bar...

Reply:
That is why we transform to - and not _ inside of ConvertToUpperCamelCase - to have nicer upper camel case identifiers as output